### PR TITLE
Add more tasks to release plan

### PR DESCRIPTION
**Pull request by danielsdeleo**
Looking for input on a few of these right now, will add comments inline.

Signed-off-by: Daniel DeLeo <dan@chef.io>

### CODE REVIEW

File: RELEASE_PLAN.md [hunk 1]
```diff
@@ -18,6 +18,35 @@ Run the full kitchen acceptance suite
...
 
+#### Recipe and Init Script Testing
+
+##### Recipe Multi-Run Test
+
+Run this recipe at least 2x so we know it works continuously
+
+```
+kitchen converge PLATFORM
+kitchen converge PLATFORM
```
Comment (danielsdeleo) [inline RELEASE_PLAN.md]: What platform(s)?

File: RELEASE_PLAN.md [hunk 2]
```diff
@@ -18,6 +18,35 @@ Run the full kitchen acceptance suite
...
+* start is run in kitchen
+* stop
+* restart
+* uninstall
+
+##### Chef Client Uninstall Detection
+
+This only applies on UNIX-like systems.
+
+* Start the agent
```
Comment (danielsdeleo) [inline RELEASE_PLAN.md]: If we don't modify `INTERVAL`, you have to wait 30m for this to occur...

File: RELEASE_PLAN.md [hunk 3]
```diff
@@ -91,3 +120,8 @@ Set the version in `lib/automate_liveness_agent/version.rb`
 
 Commit and push the version number update.
 
+#### Deploy the New Release to Chef Internal Infrastructure
+
+TODO: describe this process:
```
Comment (danielsdeleo) [inline RELEASE_PLAN.md]: I'm thinking we can document this as we do it the first time.

Comment (sdelano) [inline RELEASE_PLAN.md]: I got this

**Pull request merged**